modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -531,24 +531,23 @@ data "aws_iam_policy_document" "karpenter_controller" {
       "ec2:DescribeInstanceTypeOfferings",
       "ec2:DescribeAvailabilityZones",
       "ec2:DescribeSpotPriceHistory",
+      "ec2:RunInstances",
```

Comment:
this is already provided here https://github.com/terraform-aws-modules/terraform-aws-iam/blob/d6164989a94e1e9711e40e6fb0c374c2d3945149/modules/iam-role-for-service-accounts-eks/policies.tf#L570

Reply:
The link you provided to the policy makes reference to [karpenter v0.6.1](https://github.com/terraform-aws-modules/terraform-aws-iam/blob/d6164989a94e1e9711e40e6fb0c374c2d3945149/modules/iam-role-for-service-accounts-eks/policies.tf#L516). Karpenter is currently on v0.16.1. You may need to validate the IAM policy against the current karpenter version